test_scripts/API/SetAppIcon_backslash/ATF_N_SetAppIcon_SyncFileName_Contains_BackSlashScreened_as_Param.lua

```diff
@@ -0,0 +1,70 @@
+-- Requirement summary:
+--[GENIVI] SDL must respond INVALID_DATA in case mobile app sends "/" (backslash) symbol at "fileName" parameter
+--
+-- Description:
+-- SDL must respond INVALID_DATA in case mobile app sends "/" (backslash) symbol at "fileName" parameter" must be implemented.
+--
+-- Performed steps:
+-- Send RPC SetAppIcon with backslash screened in FileName param
+--
+-- Expected result:
+-- SDL respond with resultCode "INVALID_DATA" and success:"false"
+------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+
+--[[ Local Variables ]]
+local iTimeout = 5000
+
+--[[ Local Functions ]]
+local function check_INVALID_DATA_resultCode_OnMobile(cid)
```

Comment:
@Aleksandriys, looks unreasonable to move EXPECT_RESPONSE(cid, {success = false, resultCode = "INVALID_DATA"}) to separate Local functions block as a separate function.